src/main/modules/recaptcha.js

```diff
@@ -0,0 +1,26 @@
+import $ from 'jquery'
+
+const ReCaptcha = ($container, index) => {
+  window[`reCaptchaCallbacks_${index}`] = submitCallback($container)
+  $('button.g-recaptcha', $container).attr(
+    'data-callback',
+    `reCaptchaCallbacks_${index}`
+  )
+}
+
+const submitCallback = $el => () => {
+  $el.submit()
+}
+
+const injectReCaptchaScript = () => {
+  $('body').append(
+    '<script src="https://www.google.com/recaptcha/api.js" async defer></script>'
+  )
+}
+
+$.fn.ReCaptcha = function() {
+  $(this).each(function(index) {
```

Comment:
To clarify what `$(this)` holds respectively:
```suggestion
const $forms = $(this)

$forms.each(function (index, $form) {
  new ReCaptcha($form, index)
})
```